### ISSUE TITLE
git branch --create-reflog suggestion fails on master branch

### ISSUE THREAD

**Issue by casz** [GitTools/actions#12]
The suggestion in the readme fails on master branch

```yml
    - name: Fetch tags and master for GitVersion
    run: |
        git fetch --tags
        git branch --create-reflog master origin/master
```

**Comment by arturcic** [comment]
@casz 
can you change to 
`git fetch --prune --unshallow `
instead and see if it works?

**Comment by casz** [comment]
That requires using checkout@v2 right?

**Comment by arturcic** [comment]
yes, are you willing to send a PR?

**Comment by casz** [comment]
@arturcic of course!

**Comment by casz** [comment]
fixed #28
